networkx/classes/function.py

```diff
@@ -54,7 +54,15 @@
 
 
 def nodes(G):
-    """Returns an iterator over the graph nodes."""
+    """Returns an iterator over the graph nodes.
```

Comment:
I know this wasn't the goal of this PR, but looking over this I think this is actually incorrect - a `NodeView` object is returned, which is technically not an Iterator in the Python sense. I think this first summary line should be updated to use more precise wording; maybe something like ``Returns a view of the nodes in `G` ``.